pkgs/tools/admin/zbctl/default.nix

```
@@ -0,0 +1,37 @@
+{ lib, stdenv, fetchurl, coreutils }:
+
+stdenv.mkDerivation rec {
+  pname = "zbctl";
+  version = "8.0.6";
+
+  builder = ./builder.sh;
+
+  inherit coreutils;
+
+  src = if stdenv.hostPlatform.system == "x86_64-darwin" then fetchurl {
+    url = "https://github.com/camunda/zeebe/releases/download/${version}/zbctl.darwin";
+    sha256 = "17hfjrcr6lmw91jq24nbw5yz61x6larmx39lyfj6pwlz0710y13p";
+  } else fetchurl {
+    url = "https://github.com/camunda/zeebe/releases/download/${version}/zbctl";
+    sha256 = "1xng11x7wcjvc0vipdrqyn97aa4jlgcp7g9aw4d36fw0xp9p47kp";
+  };
+
+  meta = with lib; {
+    description = "The command line interface to interact with Camunda 8 and Zeebe";
+    homepage = "https://docs.camunda.io/docs/apis-clients/cli-client/";
+    downloadPage = "https://github.com/camunda/zeebe/releases";
+    sourceProvenance = with sourceTypes; [ binaryNativeCode ];
+    license = licenses.asl20;
+    platforms = [ "x86_64-darwin" "x86_64-linux" ];
+    maintainers = with maintainers; [ thetallestjj ];
+    longDescription =
+      ''  A command line interface for Camunda Platform 8 designed to create and read resources inside a Zeebe broker.
```

```suggestion
    longDescription = ''
      A command line interface for Camunda Platform 8 designed to create and read resources inside a Zeebe broker.
```
Looks a bit more consistent